src/views/Static/APIDoc/APIDoc.vue

```diff
@@ -298,18 +298,144 @@ puts response.read_body
     <p
       :class="['mb-4 lato-font-medium lato-text-sm',{'lato-text-md':$vuetify.breakpoint.xlOnly }]"
     >
-      The metadata section contains information about the record itself, such as its name, contact details for resource maintainers etc. The full specifications for the metadata field, in the form of the JSON schemas used to validate it, can be found at:
+      The metadata section contains information about the record itself, such as its name, contact details for resource maintainers etc. The full specifications for the metadata field, in the form of the JSON schemas used to validate it, can be found the following URLs:
     </p>
-    <a
-      class="underline-effect"
```

Comment:
please add this class back to any a link you recently have added to the page so it will look like other a tag.. it needs the underline effect when being hovered...

![image](https://user-images.githubusercontent.com/25683981/117468247-69a92480-af4c-11eb-8ca3-8b225046e1af.png)

in the above image its activated in the same page..